docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -661,19 +661,19 @@ For model_processing=*"fullaxon"*, the biophysical neuron will construct and sim
 
 Each morphology is first moved from its original coordinates in SWC to such a location that the soma is at (0, 0, 0).  Then, three rotations are applied to each morphology, in exactly the following sequence: (1) rotation around the z-axis; (2) rotation around the y-axis; (3) rotation around the x-axis.  Then, each morphology is shifted to such a location that its soma is at (x, y, z) coordinates specified in the node/node_type files.  The angles for the three rotations are also provided in the nodes/node_types files, in radians.  If a column is not provided, it is assumed that the rotation angle around that axis is 0 (that is, no rotation around that axis is applied).  The order of rotations is always the same – first around z, then around y, then around x.
 
-**morph_translate** - Optional reserved attribute, if the value is set to 0, the translation operations described above are not applied.
+**morphology** [String|Enum] - Name of the detailed morphology for a given node or node type. For name `foo`, the corresponding SWC file would be found at `$morphologies_dir/foo.swc`, where `$morphologies_dir` is specified in the [network config](#network_config).
 
-**morphology_file** - The name of the SWC file (potentially, with the path) containing the morphology for a given node or node type.
+**morph_center** - Optional reserved attribute, if the value is set to 0, morphology would _not_ be moved to (0, 0, 0) prior to rotation / translation.
```

Comment:
Add [Bool]

I don't think `morph_center` is a better name, as center could be read as a none or a verb, and in the first case is not clear it implies an action. A better name would be `recenter_morphology`, or simply `recenter`

Reply:
I started with `recenter` myself, was not sure if it was a good name.
Will bring it back then.